GetTermsAndConditionsTrigger/index.ts

```diff
@@ -0,0 +1,103 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import type { RightsToUse } from "@equinor/data-marketplace-models"
+import axios, { AxiosError } from "axios"
+import { config } from "../config"
+import { htmlToPortableText } from "../lib/html_to_portable_text"
+
+const GetTermsAndConditionTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  try {
+    const { data: relationTypeIDs } = await axios.get(`${config.COLLIBRA_BASE_URL}/relationTypes`, {
```

Comment:
I think I would consider pulling out all these calls into separate functions. Will make the main function leaner and easier to follow.